babel/dates.py

```diff
@@ -835,6 +866,100 @@ def _iter_patterns(a_unit):
     return u''
 
 
+
+def _format_fallback_interval(start, end, skeleton, tzinfo, locale):
+    if skeleton in locale.datetime_skeletons:  # If the skeleton is absolutely unknown, don't crash
+        format = lambda dt: format_skeleton(skeleton, dt, tzinfo, locale=locale)
+    else:
+        format = lambda dt: format_datetime(dt, tzinfo=tzinfo, locale=locale)
+
+    return (
+        locale.interval_formats.get(None, "{0}-{1}").
+        replace("{0}", format(start)).
+        replace("{1}", format(end))
+    )
+
+
+def format_interval(start, end, skeleton, tzinfo=None, fuzzy=True, locale=LC_TIME):
```

Comment:
Again, more tests needed, at least to cover the `tzinfo` and `fuzzy` parameters.